src/rpc/connection.rs

```diff
@@ -322,13 +322,15 @@ impl ClientConnectionAsync {
 }
 
 pub struct ServerConnection {
+    id: u32,
```

Comment:
Change to name string may be better.

Reply:
server side is passively connected, what kind of name will be proper